user_guide/intro_navigation.rst

```diff
@@ -0,0 +1,110 @@
+.. _user-guide-getting-started-controls:
+  
+Navigation Basics
+=================
+
+OroCRM provides a broad set of navigation tools that allow you to easily access necessary information and tools. The 
+system helps to ensure that the frequent actions can be performed with minimal effort by keeping required information at 
+hand. This means no important updates are missed.
+
+This section provides an overview of the navigation elements in OroCRM.
+
+
+.. _user-guide-navigation-menu:
+
+The Menu Bar
+------------
+
+The menu bar is the main mode of accessing data in OroCRM. It is always displayed at every page of OroCRM. Subject to 
+your organization configuration, it may be displayed horizontally or vertically. In the latter case, the menu items will 
+be displayed as icons.
+
+|
+
+.. image:: ./img/navigation/menu/nav_bar_top.png
+
+|
+
+.. image:: ./img/navigation/menu/nav_bar_side.png
+
+|
+
+In the mobile version the menu bar can be accessed via the "hamburger" button next to the organization name:
```

Comment:
Edit: In the mobile version, the menu bar can be accessed via the "hamburger" button next to the organization name: